product_intake_evaluation/models/res_users.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class ResUsers(models.Model):
+    _inherit = "res.users"
+
+    is_intake_evaluation_user = fields.Boolean(
+        compute="_compute_is_intake_evaluation_user",
+        inverse="_inverse_is_intake_evaluation_user",
+    )
+    is_intake_evaluation_manager = fields.Boolean(
+        compute="_compute_is_intake_evaluation_manager",
+        inverse="_inverse_is_intake_evaluation_manager",
+    )
+
+    # --- Intake Evaluation User ---
+    def _compute_is_intake_evaluation_user(self):
+        for user in self:
+            user.is_intake_evaluation_user = user.has_group(
+                "product_intake_evaluation.group_intake_evaluation_user"
+            )
+
+    def _inverse_is_intake_evaluation_user(self):
+        group = self.env.ref("product_intake_evaluation.group_intake_evaluation_user")
+        for user in self:
+            if user.is_intake_evaluation_user:
+                user.groups_id = [(4, group.id)]
```

Comment:
Please use `Command.xxx()`.
```suggestion
                user.groups_id = [Command.link(group.id)]
```